Index: src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import java.util.Scanner;\r\n\r\n//TIP To <b>Run</b> code, press <shortcut actionId=\"Run\"/> or\r\n// click the <icon src=\"AllIcons.Actions.Execute\"/> icon in the gutter.\r\npublic class Main {\r\n    public static void main(String[] args) {\r\n        Scanner input = new Scanner(System.in);\r\n\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Main.java b/src/Main.java
--- a/src/Main.java	(revision 8c00b3b3d5c9f3145bdd9de429ca07808e6a2f29)
+++ b/src/Main.java	(date 1765528076405)
@@ -1,10 +1,111 @@
+import java.util.HashSet;
+import java.util.*;
+import java.util.Scanner;
+import java.util.LinkedList;
+import java.util.Queue;
 import java.util.Scanner;
 
 //TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
 // click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
 public class Main {
-    public static void main(String[] args) {
-        Scanner input = new Scanner(System.in);
+       /* public enum Posicion {
+            DELANTERO, MEDIO, DEFENSA, PORTERO
+        }
+
+        static Set<String> nombres = new HashSet<>();
+        static Set<Posicion> posiciones = new HashSet<>();
+
+        public static void darAlta() {
+            Scanner sc = new Scanner(System.in);
+            System.out.println("como se llama el jugador?");
+            System.out.print("Nombre: ");
+            nombres.add(sc.nextLine());
+
+            System.out.print("que posicion es el jugador? ");
+            String posTexto = sc.nextLine().toUpperCase();
+
+            try {
+                Posicion pos = Posicion.valueOf(posTexto);
+                posiciones.add(pos);
+            } catch (IllegalArgumentException e) {
+                System.out.println("ERROR: esa posición no existe.");
+            }
+        }
+
+        public static void modificarJugador() {
+            Scanner sc = new Scanner(System.in);
+            System.out.println("como se llama el jugador?");
+            String nombre = sc.nextLine();
+            if (nombres.contains(nombre)) {
+                posiciones.clear();
+                System.out.print("que posicion deseas introoducir?");
+                String nuevaPos = sc.nextLine().toUpperCase();
+                posiciones.add(Posicion.valueOf(nuevaPos));
+                System.out.println("Posición modificada.");
+            } else {
+                System.out.println("El jugador no existe.");
+            }
+        }
+
+        public static void main(String[] args) {
+            Scanner input = new Scanner(System.in);
+            System.out.print("=============MENU============\n" +
+                    "1.-introducir jugador\n" +
+                    "2.-modificar jugador\n" +
+                    "3.-eliminar jugador");
+            System.out.print("que vas a introducir? ");
+            int opcion = input.nextInt();
+            switch (opcion) {
+                case 1:
+                    darAlta();
+                    break;
+                case 2:
+                    modificarJugador();
+                    break;
+            }
+*/
+       public static void main(String[] args) {
+           try {
+               int opcion;
+               do {
+                   Scanner sc = new Scanner(System.in);
+                   System.out.print("==============MENU==============\n" +
+                           "1.- Introducir producto\n" +
+                           "2.- Ver lista\n" +
+                           "3.-Introducir fecha" +
+                           "4.-salir");
+                   opcion = sc.nextInt();
+                   switch (opcion) {
+                       case 1:
+                           productos();
+                           break;
+                       case 2:
+                           verLista();
+                           break;
+                   }
+               }
+               while (opcion != 4);
+           } catch (Exception e) {
+               System.out.println(e.getMessage());
+           }
+       }
+    static Queue<String> nombreProducto = new LinkedList<>();
+    static Queue<String> fechaCaducidad = new LinkedList<>();
+
+    public static void productos() {
+        Scanner sc = new Scanner(System.in);
+        System.out.print("Nombre del producto: ");
+        String nombre = sc.nextLine();
+        nombreProducto.add(nombre);
+        System.out.print("Fecha del producto: ");
+        String fecha = sc.nextLine();
+        fechaCaducidad.add(fecha);
 
     }
-}
\ No newline at end of file
+
+    public static void verLista() {
+        Scanner sc = new Scanner(System.in);
+        System.out.print("==========LISTA==========\n" + nombreProducto + fechaCaducidad);
+    }
+}
+
Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ProjectModuleManager\">\r\n    <modules>\r\n      <module fileurl=\"file://$PROJECT_DIR$/tema3.iml\" filepath=\"$PROJECT_DIR$/tema3.iml\" />\r\n    </modules>\r\n  </component>\r\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
--- a/.idea/modules.xml	(revision 8c00b3b3d5c9f3145bdd9de429ca07808e6a2f29)
+++ b/.idea/modules.xml	(date 1765527553508)
@@ -2,6 +2,7 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
+      <module fileurl="file://$PROJECT_DIR$/ejercicio2/ejercicio2.iml" filepath="$PROJECT_DIR$/ejercicio2/ejercicio2.iml" />
       <module fileurl="file://$PROJECT_DIR$/tema3.iml" filepath="$PROJECT_DIR$/tema3.iml" />
     </modules>
   </component>
Index: ejercicio2/ejercicio2.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/ejercicio2/ejercicio2.iml b/ejercicio2/ejercicio2.iml
new file mode 100644
--- /dev/null	(date 1765527553524)
+++ b/ejercicio2/ejercicio2.iml	(date 1765527553524)
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
